### PR TITLE
MC-3533: production days and right-to-left reference added to JSON file

### PR DESCRIPTION
**Pull request by javiermotero**

### CODE REVIEW

**Comment by drewwills** [comment]
These items need to be added to the documentation (`README`). Suppliers who integrate with us rely on it.

File: src/main/java/com/mixam/mxjdf4/sdk/Component.java [hunk 1]
```diff
@@ -90,4 +91,11 @@ public void setType(ComponentType type) {
         this.type = type;
     }
 
+    public int getProductionDays() {
```
Comment (ashley-mixam) [inline src/main/java/com/mixam/mxjdf4/sdk/Component.java:94]: Can you move production days to article level please. We dont say its 7 days for the body and 3 days for the cover!

File: README.md [hunk 2]
```diff
@@ -119,6 +121,8 @@ The top level of the MxJdf document contains the following elements:
...
 |subProduct|Fine classification of the product. (optional) (SubProductType)|PRODUCT_HARD_COVER_BOOKS(1)<br>PRODUCT_PAPERBACK_BOOKS(2)<br>PRODUCT_PERFECT_BOOKLETS(3)<br>PRODUCT_WIRO_BOOKLETS(4)<br>PRODUCT_LOOP_BOOKLETS(5)<br>PRODUCT_STAPLED_BOOKLETS(6)<br>PRODUCT_MAGAZINES(7)<br>PRODUCT_CATALOGS(8)<br>PRODUCT_BOOKLETS(9)<br>PRODUCT_ZINES(10)<br>PRODUCT_COMIC_BOOKS(11)<br> PRODUCT_ART_PRINTS(12)<br>PRODUCT_MANGA(13)<br>PRODUCT_WEDDING_BOOK(14)<br>PRODUCT_YEARBOOK(15)<br>PRODUCT_COOKBOOK(16)<br> PRODUCT_COLOURINGBOOK(17)<br> PRODUCT_ARTBOOK(18)<br> PRODUCT_GRAPHIC_NOVEL(19)<br> PRODUCT_LOOKBOOK(20)<br> PRODUCT_DIARIES(21)<br> PRODUCT_JOURNALS(22)<br> PRODUCT_MENUS(23)<br> PRODUCT_CLASSIC_BOOKS(24)<br> PRODUCT_CLASSIC_PAPERBACK_BOOKS(25)<br> PRODUCT_FAMILY_HISTORY_BOOKS(26) ;|
+|rightToLeft|Field to describe if the item was written left to right(by default) or right to left |True or False |
```
Comment (ashley-mixam) [inline README.md:124]: lowercase true and false please

Comment (ashley-mixam) [reply README.md:124]: Maybe,

Indicates if the item is right to left

**Review by ashley-mixam — APPROVED** [review]

**Pull request merged**